packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,569 @@
+Since the grid is sized to the full width of its parent container, you may not want to use it for fixed width columns in a variable width container.
+
+<DoDont>
+  <Do title="Do" body="Base the Column span and offset on a 12-column layout.">
+    <Box padding="space60">
+      <Grid gutter="space30">
+        <Column span={4}>
+          <Card>
+            <Paragraph>4 Column Card</Paragraph>
+          </Card>
+        </Column>
+        <Column span={8}>
+          <Card>
+            <Paragraph>8 Column Card</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don’t use more than 12 columns within Grid.">
+    <Box padding="space60">
+      <Grid gutter="space30">
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use Grid to align content to the right or left side of a page layout.">
+    <Box padding="space60">
+      <Grid>
+        <Column span={4} offset={8}>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don’t use Grid with only one column of full-width content. This causes unnecessary elements. Box is what you’re looking for here.">
```

Comment:
This is actually better worded as "one full-width column of content".

(being super nitpicky now 😅)